microsetta_private_api/db/patches/0101.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE campaign.interested_users ADD COLUMN address_3 VARCHAR;
+ALTER TABLE campaign.interested_users ADD COLUMN address_type VARCHAR;
```

Comment:
Let's make this a boolean rather than a varchar, its purpose will be clearer.
```suggestion
ALTER TABLE campaign.interested_users ADD COLUMN residential_address BOOLEAN;

```